### DISCUSSION TITLE
Custom field from WPify

### DISCUSSION THREAD

**Discussion by ghost** [woocommerce/woocommerce#48880] [Checkout Flow]
Hello, I use plugin from WPify for ID, VAT Number for company. But in this new layout. I dont see this fields in billing fields.

**Comment by opr** [comment]
Hi @MichaelaKubasova! The new Cart and Checkout blocks were designed from the ground-up to solve many of the problems of the old checkout flow. Part of solving this problem involved re-assessing how we approach extensibility. Re-adding the mechanisms these plugins use to extend the checkout flow would likely result in the same issues re-surfacing, so plugin authors need to make their products compatible with the new blocks.

If you are facing issues with a plugin then it'd be best reaching out to the developers of that plugin directly to ask them to consider making it work with the Cart and Checkout blocks.

I'll close this discussion now as I don't think there's anything actionable from us, but you'll still be able to reply.

Thanks!
